pca/db/database_model.py

```diff
@@ -417,3 +419,22 @@ def save(self, *args, **kwargs):
         else:
             self.internal_ip_int = None  # IPv6 integer representations are too large to store in MongoDB currently
         super(ApplicationDoc, self).save(*args, **kwargs)
+
+class UserReportDoc(RootDoc):
+    """Provides schema for User Report doc."""
```

Comment:
I'd prefer to see a bit more explanation in here about what this collection and the `first_report` date actually represent, since it's not clear from the code.

Reply:
Looking good now after https://github.com/cisagov/pca-core/pull/5/commits/3733c13054a3c9a1a54dbb46983c9070043c733e and https://github.com/cisagov/pca-core/pull/5/commits/76ffea3eb11ba5c77d2593759504e71da3353c4e